protocol/avdl/chat1/local.avdl

```diff
@@ -94,20 +94,23 @@ protocol local {
   }
 
   record MessageInfoLocal {
-    boolean isNew;
     string senderUsername;
     string senderDeviceName;
   }
 
-  record Message {
+  record MessageFromServerUnboxed {
     MessageServerHeader serverHeader;
     MessagePlaintext messagePlaintext;
+  }
 
+  record MessageFromServerUnboxedWithContext {
```

Comment:
Hmm.. but that'd complicate the struct with error. Maybe we can merge `MessageInfoLocal` into `MessageFromServerUnboxed`, and rename `MessageFromServerUnboxedWithContext` into something like `MessageFromServerUnboxedWithError`. (Or is this what you had in mind?)